/**:
  ros__parameters:
    resolution_factor: 0.5
    map_topic: "map"
    mono_img_topic: "image_jpeg"
    endoded_img_topic: "image_encoded"